modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,41 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section describes the similarities and differences in these Mule versions.
+
+[%header,cols="40a,50a,50a"]
+|===
+| Feature | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
+| Policy resources | External dependencies, such as libraries or resources cannot be included in the policy and must be available in the runtime | Policies can package resources, eliminating the dependency with the runtime. |
```

Comment:
If the first column is Feature, and we are referring to policy feature, do we have to repeat "policy" in the first-column entries?
```suggestion
| External resources | Libraries and other external resources cannot be included in the policy and must be available in the runtime | Policies can package resources, eliminating dependency on the runtime. |
```

Reply:
Removed! Thanks!